src/style.css

```diff
@@ -0,0 +1,92 @@
+@font-face {
+  font-family: avenir;
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+body {
+  margin-top: 0;
+}
+
+.form {
+  position: relative;
+  width: 100%;
+  margin-top: 20px;
+  display: inline-block;
+}
+
+.form-label {
+  display: inline-block;
+  width: 100%;
+}
+
+.top-search-bar {
+  height: 70px;
+  line-height: 70px;
+}
+
+.bottom-search-bar {
+  height: 42px;
+  line-height: 42px;
+}
+
+.search-image {
+  position: absolute;
+  top: 50%;
+  transform: translateY(-50%);
```

Comment:
@iLebo-stack just a few follow-up comments
this approach is generally fine, but is not pixel-perfect given this task's test-set.
To pass tests you have to provide a pixel positioning for each image (see more comments) instead of this generic offset.
```suggestion
```